.github/workflows/kubesec.yml

```diff
@@ -0,0 +1,58 @@
+name: Kubesec
+on:
+  push: 
+    branches: [ Add-kubesec-nam ]
+  schedule:
+    - cron: '22 8 * * 4'
+
+jobs:
+  setup:
+    runs-on: ubuntu-20.04
+    permissions:
+      actions: read
+      contents: read
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Collect all yaml
+        id: list_yaml
+        run: |
+          LIST_YAML="$(find * -type f -name '*.yaml')"
```

Comment:
I wonder if it would be good to limit this a bit or exclude some paths. For example, this will now scan `examples` and `config/samples` that I'm not sure if is useful. On the other hand, we can at least be sure that it scans everything now.
I don't have a strong opinion, just wanted to mention this.

Reply:
I agree that scanning something is not a part of production doesn't improve security much. However, the example is the first thing a new consumer/consumer approaches when they start to work with this repo, and there is no better way to show the project security portrait by tightening the security level of these example code.

Reply:
Good point, let's keep it like this!